aggregators/config.go

```diff
@@ -174,6 +182,14 @@ func WithLogger(logger *zap.Logger) Option {
 	}
 }
 
+// WithOverflowLogging defines log function and aggregation interval for overflow logging.
+func WithOverflowLogging(logging OverflowLogging) Option {
```

Comment:
```suggestion
// WithOverflowLogging enables warning logs at harvest time, when overflows have occurred.
//
// Logging of overflows is disabled by default, as most callers are expected to rely on metrics
// to surface cardinality issues. Support for logging exists for historical reasons.
func WithOverflowLogging() Option {
```

WDYT? I think it's useful to log at all aggregation intervals, as overflows can happen independently for each interval. e.g. each 1 minute might not overflow, but when combined they may, and that is interesting (it may suggest some kind of dynamic keys over time).